module/data/item/equipment.mjs

```diff
@@ -215,6 +216,7 @@ export default class EquipmentData extends ItemDataModel.mixin(
     if ( actor.type === "npc" ) return 1; // NPCs are always considered proficient with any armor in their stat block.
     const config = CONFIG.DND5E.armorProficienciesMap;
     const itemProf = config[this.type.value];
+    if ( !itemProf ) return 0;
```

Comment:
Other than skipping 2 lines of code, does this give us anything? The code seems to work otherwise even if the config isn't found (`undefined === true` and `actorProfs.has(undefined)` are still valid pieces of code that return false, which ultimately results in `Number(false)` returning `0`.

Reply:
Yeah, I suppose if item prof is undefined it could still get a value from `actorProfs[this.type.baseitem]` so it's technically incorrect to return early. I'll remove it.